_rules/visible-label-in-accessible-name-2ee8b8.md

```diff
@@ -117,61 +123,261 @@ This `button` element has the text "search" rendered as an magnifying glass icon
+
+```html
+<a href="#" aria-label="ACT">
+	<p style="display: inline">A</p><p style="display: inline">C</p><p style="display: inline">T</p>
+</a>
+```
+
+#### Passed Example 10
+
+The [visible inner text][] is "Download specification".  The words "the" and "gizmo" aren't part of it.    
+
+```html
+<a aria-label="Download specification" href="#">Download <span style="visibility: hidden">the</span> <span style="display: none">gizmo</span> specification</a>
+```
+
+#### Passed Example 11
+
+The [visible inner text][] is "Download specification", which includes a space character between the two words due to the second clause of the definition of [visible inner text of a text node][].
+
+```html
+<a aria-label="Download specification" href="#"><span>Download</span><span id="space"> </span><span>specification</span></a>
+```
+
+#### Passed Example 12
+
+This example shows that the [visible inner text][] isn't always the same as the [innerText][https://html.spec.whatwg.org/multipage/dom.html#the-innertext-idl-attribute].  The visible inner text is "Download specification".  The innerText is 'Download \ngizmo\nspecification'.  This rule uses the visible inner text - not innerText.
+
+```html
+<style>
+.visually-hidden {
+    /* Source: https://www.tpgi.com/the-anatomy-of-visually-hidden/ */
+    clip-path: inset(50%);
+    height: 1px;
+    overflow: hidden;
+    position: absolute;
+    white-space: nowrap;
+    width: 1px;
+}
+</style>
+<a aria-label="Download specification" href="#">Download <span class="visually-hidden">gizmo</span> specification</a>
+```
+
+#### Passed Example 13
+
+This example shows that the [label in name algorithm][] handles many kinds of whitespace.
+
+```html
+<a aria-label="compose email" href="#">compose &nbsp;&nbsp;<br> email</a>
+```
+
+#### Passed Example 14
+
+This example passes the rule because "YYYY-MM-DD" is in brackets.  Text in brackets is removed by the [label in name algorithm][], because its not normally spoken by speech-input users.  
+
+```html
+<button aria-label="Search by date">Search by date (YYYY-MM-DD)</button>
+```
+
+#### Passed Example 15
+
+The passes for two reasons: 1) because the ellipsis ("…") is [non-text content][], and 2) because the ellipsis is neither a letter nor a digit and so is filtered out by the [label in name algorithm][].
+
+```html
+<button aria-label="Next">Next…</button>
+```
+
+#### Passed Example 16
+
+This passes because the [label in name algorithm][] effectively ignores all punctuation and emoji, in both the visible inner text and the accessible name, as long as they don't break up words.
+
+```html
+<button aria-label="💡 Submit 💡">&gt;&gt;&gt; ** Submit ** &lt;&lt;&lt;</button>
+```
+
+
```

Comment:
It's a strong example for the SC, but I can't see how to adapt it to this ACT rule.  That example (`<label for="firstname">First Name:</label> <input id="firstname" type="text" name="firstname">`) doesn't meet the applicability of this ACT rule because `<input>` doesn't support ["name from content"](https://www.w3.org/TR/wai-aria-1.2/#namefromcontent).  For us to add an example which uses that visible label "First Name:" on an element which does support "name from content" (mostly `<a>` and `<button>`) seems unrealistic to me.  Suggestions welcome on how to reconcile this.